### DISCUSSION TITLE
I dont understand how are the price of Image optimization with Vercel ?

### DISCUSSION THREAD

**Discussion by GabVlado** [vercel/community#1861] [Help] [answered]
What happen when 10000 users use my App? .  If  I use SSG  then 10000 users X Number of Images of the page ??. Or the price is 10000 users X 1 Once Unique Optimization  of the Page ?.

**Comment by amyegan** [comment, marked as answer]
When using Image Optimization, you are billed for the number of [source images](https://vercel.com/docs/concepts/image-optimization#source-images) requested during a billing period. The usage count is determined by the `src` prop.

> Since images are optimized on-demand, the source image is not counted until a request is made to optimize it at runtime. This means that usage can be lower if no one visited a page with an image in a given billing period. It also means there is an upperbound of source images which can be derived by the number of unique src props for all next/image components in your application.

Beyond the first request, the number of subsequent visitors would not increase the image count unless unique `src` values were used for each visitor. You can [minimize your Image Optimization costs](https://vercel.com/docs/concepts/image-optimization/managing-image-optimization-costs#how-to-minimize-image-optimization-costs) by limiting variation in the `src` prop.

You can find more information in our [Image Optimization](https://vercel.com/docs/concepts/image-optimization#) documentation, including [usage limits](https://vercel.com/docs/concepts/image-optimization/limits-and-pricing#) and strategies to [manage Image Optimization usage and costs](https://vercel.com/docs/concepts/image-optimization/managing-image-optimization-costs#)
